### DISCUSSION TITLE
Staking Fetch

### DISCUSSION THREAD

**Discussion by bradsonchloe** [fetchai/uAgents#221] [Q&A] [answered]
Hi, I just moved my Fetch from Metamask to the Fetch wallet, and tried to stake. I did not see a staking option. So I then tried to bridge my ERC-20 fet to the mainnet token hoping this would correct the issue. I paid gas for the bridging transaction but from what I can see my tokens are still ERC-20. Why did my bridge attempt fail even though I was charged a gas fee for the operation? And, will the staking option become available in the wallet if I can successfully bridge to the mainnet?

**Comment by kshipra-fetch** [comment]
Hey @bradsonchloe 

Thank you for your query. @NikoDMT, can you please have a look at this one?

**Reply by NikoDMT** [reply, marked as answer]
Bridging from erc20 to native using the token bridge ( https://token-bridge.fetch.ai/) can take a bit of time (anywhere between 1 and 24 hours). The bridge is a smart contract and as long as you've provided the correct destination address your funds will always arrive. 

Staking is only possible with native FET on the Fetch.ai network. Once you see your bridged funds you can follow this guide to finalize the staking process. https://fetch.ai/docs/guides/fetch-network/how-to-stake#staking
